tests/test_spike.py

```diff
@@ -75,3 +84,22 @@ def test_bar_chart_8(self):
             spike([1, 2, 3, 4, 5, 6, 7, 8]),
             u'\u2581\u2582\u2583\u2584\u2585\u2586\u2587\u2588'
         )
+
+    def test_bar_chart_9(self):
+        """
+        should return bar-chart -><-
+        """
+        self.assertEqual(
+            spike([.1, ], 0),
+            ''
+        )
+
+    def test_bar_chart_10(self):
+        """
+        should return bar-chart -><-
+        """
+        self.assertEqual(
+            spike([0, ], 1),
+            ' '
+        )
+
```

Comment:
__[flake8]__
* [W391] blank line at end of file

<!-- Comment by [SideCI](https://sideci.com) -->